src/fhir/input/fsh/profiles/VSDMPatient.fsh

```diff
@@ -0,0 +1,57 @@
+Profile: VSDMPatient
+Parent: Patient
+Id: vsdm-patient
+* ^url = "https://gematik.de/fhir/vsdm2/StructureDefinition/VSDMPatient"
+* insert Meta
+* meta 1..1
+* meta.profile 1..1
+* meta.profile = "https://gematik.de/fhir/vsdm2/StructureDefinition/VSDMPatient" (exactly)
+* identifier 1..1
+* identifier ^slicing.discriminator.type = #pattern
+* identifier ^slicing.discriminator.path = "type"
+* identifier ^slicing.rules = #closed
+* identifier contains
+    versichertenId_GKV 0..1 and
+    versichertenID_pkv 0..1
```

Comment:
s.o. PKV Identifier braucht es mE nicht mehr

Reply:
Punkt wird mit in die Diskussion genommen